### PR TITLE
Expand Sidebar

### PR DESCRIPTION
**Pull request by Bgreen23**
This PR adds expand/hide to all left nav items.

Docs preview: https://5fcea4bc8947a221192ba1ca--shiftlab-pytorch-docs.netlify.app/
Tutorials preview: https://5fcea930b8fb0127eb6b5ecf--shiftlab-pytorch-docs.netlify.app/

### CODE REVIEW

File: js/theme.js [hunk 1]
```diff
@@ -264,26 +264,29 @@ if (downloadNote.length >= 1) {
 
 //This code makes the Notes section of the Docs Left Nav collapsible
```
Comment (patmellon) [inline js/theme.js:265]: Can you update this note to reflect your changes?

**Review by patmellon — CHANGES_REQUESTED** [review]
It looks like some of the link titles are being cut off:

Screen Shot 2020-12-08 at 4 21 33 PM

Can you make sure that you include the build files for any PR for this repo?

File: js/theme.js (hunk 1 above)
Comment (Bgreen23) [reply js/theme.js]: > It looks like some of the link titles are being cut off:
> 
> Screen Shot 2020-12-08 at 4 21 33 PM
> 
Is this still causing an issue? I cleared my cache and it is showing that the link is separated on 2 lines

**Review by patmellon — APPROVED** [review]
This looks good. You might have to update your build file again, though. I pushed up the latest commits from upstream/master.

**Pull request closed without merging**